cmdx.py

```diff
@@ -729,15 +734,26 @@ def isA(self, type):
             False
 
         """
+        if isinstance(type, om.MTypeId):
+            return type == self._fn.typeId
+        elif isinstance(type, string_types):
+            return type == self._fn.typeName
+        elif isinstance(type, (tuple, list)):
+            return self._fn.typeName in type or self._fn.typeId in type
+        elif isinstance(type, int):
+            return self._mobject.hasFn(type)
 
-        return self._mobject.hasFn(type)
```

Comment:
We should warn here, the user might not understand types don't match because of an invalid type.

```py
else:
  cmds.warning("Unsupported argument passed to isA('%s')" % type)
  return False
```